htmlbasic.go

```diff
@@ -205,6 +211,9 @@ func (html *HTMLBasicType) Write(lineHt float64, htmlStr string) {
 			case "center":
 				html.pdf.Ln(lineHt)
 				alignStr = "L"
+			case "right":
```

Comment:
Adding support for the 'right' closing html tag. Fallbacking to 'left' alignment after close.